db/migrate/20221129094342_create_value_definitions.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Data model for storing value definitions
+class CreateValueDefinitions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :values, id: :uuid do |t|
+      t.string :ref_id
+      t.string :title
+      t.text :description
+      t.string :value_type
+      t.numeric :lower_bound, array: true, default: nil
+      t.numeric :upper_bound, array: true, default: nil
```

Comment:
According to the XCCDF specs, lower/upper bounds are "decimals" which is a float and since we're only worrying about the default upper/lower bounds for now I don't think this needs to be an array.